join/early_adopter.md

```diff
@@ -143,6 +143,10 @@ the platform through any of the clients. With all clients you will need to conne
 `https://openeo.cloud` and then authenticate through EGI check-in with the 
 account used above.
 
+::: tip Tip
+We advise you to always use the same identity provider  you originally signed up with when identifying yourself through EGI Check-in (as this is the one associated with your EGI account authorized by the platform). If you want to use a different provider than the one you originally signed up with, you must first link the new account to your existing EGI account. Read more about how to link different identity providers to your EGI account in the EGI documentation [here](https://docs.egi.eu/users/aai/check-in/linking/).
```

Comment:
The first sentence is a bit hard to parse as intended I think

suggestion:

> We advise you, for your own ease of use, to always log in with the same identity provider you originally registered with. Otherwise you risk ending up with a separate new EGI account, that again has to go through the openEO Platform virtual organization acceptance flow.
> It is however possible to link multiple accounts from multiple identity providers to the same EGI account, but this has to happen before you use these accounts to log in as explained in the [EGI documentation](https://docs.egi.eu/users/aai/check-in/linking/).

Reply:
Thanks for the suggestion. I rearranged the word order a bit but otherwise copied it.

Reply:
great!